proto/penumbra/penumbra/core/component/funding/v1/funding.proto

```diff
@@ -7,6 +7,27 @@ import "penumbra/core/component/shielded_pool/v1/shielded_pool.proto";
 import "penumbra/core/keys/v1/keys.proto";
 import "penumbra/core/num/v1/num.proto";
 import "penumbra/crypto/decaf377_rdsa/v1/decaf377_rdsa.proto";
+import "penumbra/core/txhash/v1/txhash.proto";
+
+// Query operations for the funding component.
+service FundingService {
+  // Checks if a particular nullifier has already been used in the current epoch.
+  rpc LqtCurrentEpochVoted(LqtCurrentEpochVotedRequest) returns (LqtCurrentEpochVotedResponse);
+}
+
+// Request if a nullifier is already used in the current epoch.
+message LqtCurrentEpochVotedRequest {
+  // The nullifier to check.
+  sct.v1.Nullifier nullifier = 1;
```

Comment:
I would add an `epoch_index`. If it's present, check the specific epoch. If it's absent, check the current one.